<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>125</width>
    <height>82</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>Memory Address Register</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="_label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MAR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="_buttonSet">
          <property name="toolTip">
           <string>Set the MAR</string>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WidgetLed" name="_ledRun" native="true">
          <property name="toolTip">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WidgetLed" name="_ledSet" native="true">
          <property name="toolTip">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="WidgetLedBank" name="_widgetLeds" native="true"/>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="WidgetDip" name="_widgetDip" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetLedBank</class>
   <extends>QWidget</extends>
   <header>widgetledbank.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetDip</class>
   <extends>QWidget</extends>
   <header location="global">widgetdip.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLed</class>
   <extends>QWidget</extends>
   <header location="global">widgetled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
